projects/cloud/app/frontend/entrypoints/app.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-
+import RootStore from "../stores/RootStore";
 import App from '@/components/App';
 
+const store = new RootStore();
```

Comment:
I think we can pass this one down through a context, and components can have component-level store that wraps the domain-specific stores. Those component-level stores would act as interactions in a VIPER architecture.